<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Perfil</class>
 <widget class="QWidget" name="Perfil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_usuario">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cuadroblanco">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>371</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border-width: 5px;
    border-style: solid;
    border-radius: 15px;
	border-color: #646464;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_foto">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-width: 2px;
border-style: solid;
	border-color: gray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>1271</width>
     <height>941</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{
	border-width: 2px;
	border-style: solid;
	border-color: gray;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_correo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #A6A6A6;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_seguidores">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>310</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(255,255,255,0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_seguidos">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>310</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(255,255,255,0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_seguidore1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(255,255,255,0);
}</string>
   </property>
   <property name="text">
    <string>Seguidores</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_seguidos1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(255,255,255,0);
}</string>
   </property>
   <property name="text">
    <string>Seguidos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_seguir">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #086FE6;
color: white;
border-style: solid;
border-color: blue;
border-radius: 40px;
}</string>
   </property>
   <property name="text">
    <string>Seguir</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_CuadroBuscar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border-width: 3px;
    border-style: solid;

	border-color: #646464;
	background-color: white;
}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_ordenar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Ordenar por:</string>
   </property>
  </widget>
  <zorder>lbl_cuadroblanco</zorder>
  <zorder>lbl_usuario</zorder>
  <zorder>lbl_foto</zorder>
  <zorder>tableWidget</zorder>
  <zorder>lbl_correo</zorder>
  <zorder>bt_seguidores</zorder>
  <zorder>bt_seguidos</zorder>
  <zorder>bt_seguidore1</zorder>
  <zorder>bt_seguidos1</zorder>
  <zorder>bt_seguir</zorder>
  <zorder>lbl_CuadroBuscar</zorder>
  <zorder>cb_ordenar</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
